appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockButtonBase.java

```diff
@@ -233,7 +233,7 @@ private void setFontTypefaceProperty(String text) {
    */
   private void setImageProperty(String text) {
     imagePropValue = text;
-    String url = convertImagePropertyValueToUrl(text);
+    String url = MockComponentsUtil.convertAssetValueToUrl(editor, text);
```

Comment:
Why do we need to modify this here?

Reply:
Because we've removed the `convertImagePropertyValueToUrl` method from `MockComponent`. It did the same thing as `MockComponentsUtil.convertAssetValueToUrl`.